app/api/helpers/export_helpers.py

```diff
@@ -29,8 +29,8 @@
 FIELD_ORDER = {
     'event': [
         'id', 'name', 'latitude', 'longitude', 'location_name', 'starts_at', 'ends_at',
-        'timezone', 'description', 'original_image_url', 'logo_url', 'organizer_name',
-        'organizer_description', 'external_event_url', 'ticket_url', 'privacy', 'event_type_id',
+        'timezone', 'description', 'original_image_url', 'logo_url', 'owner_name',
```

Comment:
@shreyanshdwivedi IMO, organizer name should be preserved too for the sake of upcoming roles or we are using only owner for now?

Reply:
@uds5501 we can get details of organizers via Relationship we have. Only owner info should suffice here as it can be used for filtering too.